Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -586,6 +586,31 @@ public extension String {
     }
     #endif
 
+    #if canImport(Foundation)
+    // https://www.hackingwithswift.com/articles/141/8-useful-swift-extensions
+    /// SwifterSwift: Replaces a set number of occurrences of a string with replacement text.
+    ///
+    ///        "I swift swift".replacingOccurrences(of: "swift", with: "love", count: 1) -> "I love swift."
+    /// - Parameters:
+    ///     - search: the string to be replaced
+    ///     - replacement: the replacement string
+    ///     - maxReplacements: the maximum number of times to replace the string, which must be greater than 0
+    /// - Returns: The new string with the replacement text swapped the specified amount of times.
+    func replacingOccurrences(of search: String, with replacement: String, count maxReplacements: Int) -> String {
+        guard maxReplacements > 0 else { return self }
+
+        var count = 0
+        var returnValue = self
+
+        while let range = returnValue.range(of: search), count < maxReplacements {
```

Comment:
Reverse these to avoid performing an extra search on the string at the end of the count. It's a tiny optimisation but why not? 😄 
```suggestion
        while count < maxReplacements,
           let range = returnValue.range(of: search) {
```